server/routes/bookedtoday/arrivals/singleRecordFoundController.ts

```diff
@@ -1,17 +1,28 @@
 import type { RequestHandler } from 'express'
 import { State } from './state'
 import type { ExpectedArrivalsService } from '../../../services'
+import { convertToTitleCase } from '../../../utils/utils'
 
 export default class SingleRecordFoundController {
   public constructor(private readonly expectedArrivalsService: ExpectedArrivalsService) {}
 
   public view(): RequestHandler {
     return async (req, res) => {
       const { id } = req.params
-      const existingArrival = State.newArrival.get(req)
-      const perRecord = await this.expectedArrivalsService.getArrival(id)
+      const data = await this.expectedArrivalsService.getArrival(id)
 
-      return res.render('pages/bookedtoday/arrivals/singleMatchFound.njk', { id, perRecord, existingArrival })
+      const match = data.potentialMatches[0]
+      State.newArrival.set(res, {
```

Comment:
This is fine for now, we need to change the link on this page to a button and set the new arrival would be on the form submit - so we only set it once we confirm the arrival